### PR TITLE
[ENG-1436] Sloan Author assertions

### PR DESCRIPTION
**Pull request by fabmiz**
## Purpose

Display sloan author assertions on preprint detail.

## Summary of Changes/Side Effects
- Add assertions components
- Install and use `ember-basic-dropdown` addon for assertions dropdown
- Move preprint edit button, delete preprint-provider-logo, move preprint timestamps
to below the preprint file iframe. Supplemental materials link is now in the preprint metadata sidebar on the right.
- Add analytics
## Ticket

[ENG-1436](https://openscience.atlassian.net/browse/ENG-1436)

## Notes for Reviewer

# Reviewer Checklist

- [ ] meets requirements
- [ ] easy to understand
- [ ] DRY
- [ ] testable and includes test(s)
- [ ] changes described in `CHANGELOG.md`

### CODE REVIEW

File: app/templates/components/author-assertions.hbs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+        {{#if this.shouldShowPreregLinks}}
+            <div class='assertions-container'>
+                {{#assertions-dropdown
+                    available=preprint.hasPreregLinks
+                    assertionName=(t 'components.author-assertions.prereg.title')
+                    as |dropdown|
+                }}
+                    {{#dropdown.content}}
+                        {{#if (eq preprint.hasDataLinks 'available')}}
```
Comment (aaxelb) [inline app/templates/components/author-assertions.hbs]: should use `hasPreregLinks`

File: app/locales/en/translations.js [hunk 2]
```diff
@@ -49,18 +49,21 @@ export default {
...
+            submitted_on: 'Submitted',
         },
         share: {
             download: 'Download',
             downloads: 'Downloads',
             download_file: 'Download file',
             download_preprint: 'Download {{documentType.singular}}',
         },
+        'author-assertions': {
+            title: 'Author Assertions',
```
Comment (aaxelb) [inline app/locales/en/translations.js]: move this to join `components.author-assertions`

File: app/locales/en/translations.js [hunk 3]
```diff
@@ -474,6 +477,24 @@ export default {
                 base: 'This {{documentType.singular}}',
             },
         },
+        'author-assertions': {
+            no: 'No',
+            yes: 'Yes',
+            conflict_of_interest: {
+                title: 'Conflict of Interest',
+                no: 'Author asserted no Conflict of Interest',
```
Comment (aaxelb) [inline app/locales/en/translations.js]: maybe check if this is the text we want (it matches the mockup but is inconsistent with the other text)

**Comment by fabmiz** [comment]
@aaxelb fixed.

**Pull request merged**